packages/odf/components/topology/sidebar/storage-cluster/StorageClusterDetails.tsx

```diff
@@ -138,7 +139,7 @@ export const StorageClusterDetails: React.FC<StorageClusterDetailsProps> = ({
                 isLoading={!nodesLoaded}
                 error={!!nodesLoadError}
                 kind={NodeModel as any}
-                resources={getCephNodes(nodesData as K8sResourceKind[])}
+                resources={getCephNodes(nodesData as NodeKind[])}
```

Comment:
```suggestion
                resources={getCephNodes(nodesData)}
```
Add type to `watchK8sResource`